app/src/main/java/com/github/kulebin/myfoursquareapp/http/HttpClient.java

```diff
@@ -111,4 +124,22 @@ private void execute(final String pUrl, final HttpRequestType pRequestType, fina
             pIOnResult.onError(exception);
         }
     }
+
+    private boolean isConfigured() {
+        return mInterceptor != null;
+    }
+
+    private <T> T modifyRequest(final T pRequest) {
```

Comment:
modifyRequest() method refactored, only HttpRequest is gotten as a param and returned as a result